_scripts/migrations/0028_notification_settings.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE discussion_user_access
+    ADD COLUMN IF NOT EXISTS notif_setting varchar(10) DEFAULT 'EVERYTHING' not null;
```

Comment:
I think this is going to fail and require a two step operation. First add without not null, then migrate, then add not null.